ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,60 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+    """
+
     api_group = Resource.ApiGroup.MACHINECONFIGURATION_OPENSHIFT_IO
 
     class Status(Resource.Status):
         UPDATED = "Updated"
         UPDATING = "Updating"
+
+    def __init__(
+        self,
+        name=None,
+        labels=None,
+        machine_config_selector_matches=None,
```

Comment:
```suggestion
        machine_config_selector=None,
```
use the attribute names as much as possible; the docstring should describe what this attribute is for

Reply:
Ack